wm.c

```
@@ -1,20 +1,37 @@
 #include <Windows.h>
 #include <signal.h>
 #include <stdlib.h>
+#include <stdio.h>
 
 HHOOK hookHandle;
 
 void ctrlc(int sig) {
-    UnhookWindowsHookEx(hookHandle);
+    if (!UnhookWindowsHookEx(hookHandle)) {
+        wprintf(L"[!] Error while unhooking hookHandle.\nError: %d\n", GetLastError());
+    }
+    
     exit(0);
 }
 
 int main() {
     HMODULE wmDll = LoadLibraryW(L"wm_dll");
+
+    if (wmDll == NULL) {
+        wprintf( L"[!] Error while loading library.\n Error: %ld", GetLastError());
```

Thanks for the contribution! Could you also return an error?